src/debug.cpp

```diff
@@ -280,8 +280,8 @@ time_info get_time() noexcept {
     timeval tv;
     gettimeofday( &tv, nullptr );
 
-    auto const tt      = tv.tv_sec;
-    auto const current = localtime( &tt );
+    time_t const tt      = tv.tv_sec;
+    auto   const current = localtime( &tt );
```

Comment:
In fact, you could omit this altogether, or turn the first line into `auto const tt = time_t { tv.tv_sec };` if you're absolutely sure you need the conversion (if so, explain why).